docs/quickstart.rst

```diff
@@ -44,13 +42,6 @@ A locust file is just a normal Python module, it can import code from other file
 
 The behaviour of a simulated user is represented by a class in your locust file. When you start a test run, Locust will create an instance of the class for each concurrent user.
 
-.. code-block:: python
-
-    wait_time = between(1, 2)
-
-The class defines a ``wait_time`` that will make the simulated users wait between 1 and 2 seconds after each task (see below)
-is executed. For more info see :ref:`wait-time`.
-
```

Comment:
Wait time for Locust users its a really important concept when using Locust as originally intended (defining user behaviour in code, and then choosing how many simultaneous users using a website to simulate), and I definitely think it should exist in the quickstart documentation.

There are definitely use-cases for zero or super low wait time as well, but I don't think that's a reason to remove this short paragraph from the introduction docs.

Reply:
I am aware of your arguments, and you are of mine so I wont repeat myself. The community voted 3/4 in favour of changing it in https://github.com/locustio/locust/issues/1308

Reply:
> The community voted in favour of changing it in #1308

"The community" did not vote wether to include wait time in the quick start documentation.

Reply:
I also think having community votes to decide these things is a bad idea, and it's nothing that we've decided to have. 

Out of curiosity - and since I didn't recognize the usernames of most of those who had "voted" - I checked the number of commits they had done to the Locust project. Among those who had voted 👍 , besides you, only one other user (though two were hidden behind "2 more" by the Github UI) had a single commit which was a typo fix. Among the 4 who had voted 👎 , besides me one had multiple (although old) commits and one had a single small fix commit.

Having "community votes" to decide design disagreements is a really bad idea in my opinion, and definitely not something that we've decided to have.

Reply:
And out of the people who have taken their time to actually *comment* on the issue with arguments for the issue in question, *none* of them were arguing for defaulting the wait time to zero.

Reply:
You haven't really been around for so long, so I did not expect you to care, and when I saw the large support for the issue I went ahead and implemented it.

I dont agree that the votes supporting your point of view are worth 3 times the ones supporting mine. You interpret the lack of comments as a lack of support, when it is just as likely that they just found that my argumentation was reasonable and they had nothing more to add. Would it have been better if I had phrased myself less convincingly so that other people would fill in the gaps, thus adding to the comment count?

I agree that community votes to decide design disagreements is not good, but as a last resort I think it is better than nothing when the two people with the biggest "claim" on the project cannot agree (and tbh, our energy is better spent almost anywhere else :)

If we are going to continue discussing this, can I suggest we appoint some kind of neutral arbiter? Someone we both think is a reasonable person (maybe some contributor that has not yet weighed in on this issue), and whose decision we can both accept.

Reply:
> You haven't really been around for so long, so I did not expect you to care

Well I do, and I have actively participated in arguing the issue in #1308.

> I dont agree that the votes supporting your point of view are worth 3 times the ones supporting mine.

Like I said, I think having random votes decide design decisions is a really bad idea. I only commented on the vote because you brought it up. If I was to draw any conclusion from #1308, it would be that the only two people caring much about the issue is you and I.

> You interpret the lack of comments as a lack of support, when it is just as likely that they just found that my argumentation was reasonable and they had nothing more to add

I really don't know how to interpret a bunch of thumbs up from random users, on a comment that only stated the arguments from one side of the issue. I would have liked for some counter-arguments to the arguments brought up by me. I think that would have been much more constructive.

> and tbh, our energy is better spent almost anywhere else :)

I agree.

> If we are going to continue discussing this, can I suggest we appoint some kind of neutral arbiter? Someone we both think is a reasonable person (maybe some contributor that has not yet weighed in on this issue), and whose decision we can both accept.

I think it would be great to hear the opinion of more contributors who haven't weighed in. Ideally, one of us could be convinced of the other's perspective, though I realize that might be hoping on too much :).

In the meantime, would you be okay with at least including `wait_time` in the Quickstart documentation? I think it's a very important aspect of Locust, and I'm convinced it makes the documentation better. I get that you don't think so, but maybe you could live with a small piece of "unnecessary" info in the introduction docs?

Reply:
>  really don't know how to interpret a bunch of thumbs up from random users, on a comment that only stated the arguments from one side of the issue. I would have liked for some counter-arguments to the arguments brought up by me. I think that would have been much more constructive.

Maybe all users did not read and understand the implications of the issue, but I think it is more likely than not that that people did read the ticket, as well as followup reasoning by you and me (at least that is how I interpret the the 6 upvotes for my followup argument & clarification and Anuj's argument that got 3 upvotes).

> In the meantime, would you be okay with at least including wait_time in the Quickstart documentation? I think it's a very important aspect of Locust, and I'm convinced it makes the documentation better. I get that you don't think so, but maybe you could live with a small piece of "unnecessary" info in the introduction docs?

I think the simplicity of beginner documentation is one of the key reasons for not forcing people to be explicit about wait times. Particularly I feel that forcing new users to learn & think about wait times _before_ thinking about tasks is the wrong order for useability/pedagogy (remember that I wanted to move wait_time to the end of the class in that particular example? you do have a good point about keeping attributes before functions though...) 

But if you feel this strongly about it then yes, for the sake of moving on, you can put it back into quickstart. Maybe one of us will turn around completely later on, who knows :)

Sorry if come across as a stubborn idiot :)